### DISCUSSION TITLE
Postcss with Nuxt 3.0.0

### DISCUSSION THREAD

**Discussion by Katerinka28** [nuxt/nuxt#16349] [Questions]
Ah, yes - we do not respect your `postcss.config.js` file - you need to set options directly in `nuxt.config`.

_Originally posted by @danielroe in https://github.com/nuxt/framework/issues/9163#issuecomment-1320018577_
      

I`am used Nuxt3 rc 4 . Postcss i add with this options in nuxt.config.js

```
 build: {
    postcss: {
      postcssOptions: require(resolve(__dirname, "postcss.config.js")),
    },
```

After upgrade Nuxt to official release(3.0.0) postcss doesn`t  work.
In docs build.postcss is absent.

Can you help me? Where is the right place to set postcss options in nuxt.config.js???

**Comment by danielroe** [comment]
in the `postcss` top-level config option. You can set `config` and `plugins` under it.
